packages/lwt/lwt.4.3.1/opam

```diff
@@ -0,0 +1,65 @@
+opam-version: "2.0"
+
+synopsis: "Promises and event-driven I/O"
+
+version: "4.3.1"
```

Comment:
```suggestion
```
Can we remove the redundant field or it is necessary for other mechanism. I remember there was some dune-related issue some times ago

Reply:
It's necessary for Dune to fill the version field in `META`, which is then shown in ocamlfind query, which is still used by some package installation scripts. https://github.com/ocsigen/lwt/issues/690